### PR TITLE
CSA-52, bugfix CSA-50

### PR DESCRIPTION
**Pull request by tymnik**
- registration redux created, page added and styled;
- edit profile page added;

### CODE REVIEW

File: src/components/RegistrationForm/RegistrationForm.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+              className={css.customFileButton}
+              onClick={handleFileButtonClick}
+            >
+              Upload
+              <span>
+                <svg className={css.uploadIcon}>
+                  <use xlinkHref={`${sprite}#upload`} />
+                </svg>
+              </span>
+            </button>
```
Comment (VovaLyapota) [inline src/components/RegistrationForm/RegistrationForm.tsx]: Here you'd better use an Button component with Icon component in "icon" props

File: src/components/UserNav/UserNav.tsx [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+          <svg className={css.userIcon}>
+            <use xlinkHref={`${sprite}#user`} />
+          </svg>
+        )}
+        <p className={css.userName}>{user.name || "Mike"}</p>
+      </div>
+      <Link to={ROUTES.PROFILE} className={css.arrowLink}>
+        <svg className={css.arrowIcon}>
+          <use xlinkHref={`${sprite}#arrow-down`}/>
+        </svg>
```
Comment (VovaLyapota) [inline src/components/UserNav/UserNav.tsx]: Here also need an Icon component instead

File: src/components/RegistrationForm/RegistrationForm.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import css from './RegistrationForm.module.css';
+
+interface FormData {
+  name: string;
+  email: string;
+  password: string;
+  confirm_password: string;
+  avatar?: File | undefined;
+}
+
```
Comment (StanislavKharkiv) [inline src/components/RegistrationForm/RegistrationForm.tsx]: It's a pretty big component, so better move types in other file

File: src/pages/EditProfilePage/EditProfilePage.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import css from './EditProfilePage.module.css'
+
+const EditProfilePage = () => {
+    const handleSelect = (value: string) => {
+        console.log('Selected value:', value);
+    };
+    const user = useSelector(selectUser);
+
+    return (
+        <section>
```
Comment (StanislavKharkiv) [inline src/pages/EditProfilePage/EditProfilePage.tsx]: I think better always write hooks first and then our functions and variables

File: src/pages/RegistrationPage/RegistrationPage.module.css [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  background-color: rgba(10, 102, 194, 1);
+}
+
+.linkedinButton:hover,
+.linkedinButton:focus,
+.linkedinButton:active {
+  border-color: rgb(13, 86, 160);
+  background-color: rgb(13, 86, 160);
+}
+
```
Comment (StanislavKharkiv) [inline src/pages/RegistrationPage/RegistrationPage.module.css]: we should decide how to write colors: hex or rgb (i don't mean rgb with alfa channel rgba())

File: src/components/ProfileSelector/ProfileSelector.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+import sprite from '../../images/sprite.svg'
+
+import css from './ProfileSelector.module.css';
+
+interface SelectorProps {
+    onSelect: (value: string) => void;
+}
+
+const Selector: React.FC<SelectorProps> = ({ onSelect }) => {
+    const [selectedOption, setSelectedOption] = useState<string>('profile');
```
Comment (StanislavKharkiv) [inline src/components/ProfileSelector/ProfileSelector.tsx]: I thought we are decided don't write return types for Components

**Review by VovaLyapota — APPROVED** [review]

**Review by StanislavKharkiv — APPROVED** [review]

**Pull request merged**